third_party/move/move-compiler-v2/tests/control-flow-simplification/jump-label.off.exp

```diff
@@ -0,0 +1,10 @@
+
+Diagnostics:
+error: duplicate declaration, item, or annotation
```

Comment:
Pretty sure producting this error is not the intention of this test. Please fix.

Reply:
done